packages/core/src/modules/vc/data-integrity/models/W3cJsonLdVerifiableCredential.ts

```diff
@@ -55,4 +60,8 @@ export class W3cJsonLdVerifiableCredential extends W3cCredential {
   public get encoded() {
     return this.toJson()
   }
+
+  public get jsonCredential(): W3cJsonCredential {
```

Comment:
Can't the `this.encoded` method just be used for this? Or is the casting something that should stay unique for this method

Reply:
I want to have the same methods availalbe on both JWT and JSON-LD as otherwise you have to do instance checks. This way typescript is smart enough to see `jsonCredentail` is availalbe on both.

While encoded is string for JWT and object for JSON-LD.

I'm also thinking of adding `encoded` to SdJwtVc in AFJ as that will allow you to call .encoded no matter if you have an SD-JWT, JWT or JSON-LD, making it a lot easier to work with

Reply:
Ah I see, that makes sense!